modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-values-factory/src/main/java/com/liferay/dynamic/data/mapping/form/values/factory/internal/DDMFormValuesFactoryImpl.java

```diff
@@ -108,17 +108,24 @@ private void _checkDDMFormFieldParameterNames(
 			return;
 		}
 
+		HashMap<String, DDMFormField> fullDDMFormFields = new HashMap<>();
+
```

Comment:
@linolaoj can we change to `Set` instead of `Map`?

Reply:
He @carolmariaabb, I used HashMap because I thought it would perform better here. But I can test if HashSet performs better and change it.